services/db/recovery.go

```diff
@@ -197,6 +220,19 @@ func HandleRecovery(forceNormal bool, forceDegraded bool, forceMaintenance bool,
 		}
 	}
 
+	//TODO recoveryModeStat
+	recoveryModeStat := stats.NewStat("recovery.mode_normal", stats.GaugeType)
+	if recoveryData.Mode != normalMode {
```

Comment:
remove these lines, as it's being done in sendRecoveryModeStat goroutine below ?